templates/quickstart-bitbucket-dc.template.yaml

```diff
@@ -165,6 +173,24 @@ Parameters:
     ConstraintDescription: 'Must be a valid Bitbucket version number. For example: 5.0.0 or higher'
     Description: 'Version of Bitbucket to install. Please use version 5.0.0 or higher. Find valid versions at http://go.atlassian.com/bbs-releases'
     Type: String
+  BitbucketLicenseKey:
+    Default: ''
+    Description: Provide a license key for Bitbucket Data Center if you have one. 
+    Type: String
+  BitbucketAdminPassword:
```

Comment:
Do we really want to force the user to supply this? Can it be made optional?

Reply:
It has been made optional

Reply:
I think you need to provide `Default: ''` to make it optional.

Reply:
```AllowedPattern: '((?=^.{6,255}$)((?=.*\\d)(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[^A-Za-z0-9])(?=.*[a-z])|(?=.*[^A-Za-z0-9])(?=.*[A-Z])(?=.*[a-z])|(?=.*\\d)(?=.*[A-Z])(?=.*[^A-Za-z0-9]))^.*)|(^$)'```